### PR TITLE
Added note on failover datacenter location

### PR DESCRIPTION
**Pull request by FilippSe**
Added note on failover datacenter location

### CODE REVIEW

File: Teams/direct-routing-trunk-failover-on-outbound-call.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ This topic describes how to avoid trunk failovers on outbound calls--from Teams
 
 ## Failover on network errors
 
-If a trunk cannot be connected for any reason, the connection to the same trunk will be tried from a different Microsoft Datacenter. A trunk might not be connected, for example, if a connection is refused, if there is a TLS timeout, or if there are any other network level issues.
+If a trunk cannot be connected for any reason, the connection to the same trunk will be tried from a different Microsoft Datacenter. Note that the datacenter might be located in a different geographical region, outside of your current one (e.g. outside of North America). A trunk might not be connected, for example, if a connection is refused, if there is a TLS timeout, or if there are any other network level issues.
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-trunk-failover-on-outbound-call.md:27]:
```suggestion
If a trunk cannot be connected for any reason, the connection to the same trunk will be tried from a different Microsoft datacenter. Note that the datacenter might be located in a different geographical region, outside of your current region (for example, outside of North America). A trunk might not be connected, for example, if a connection is refused, if there is a TLS timeout, or if there are any other network-level issues.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @FilippSe, kindly commit the suggested changes. Many thanks.

**Comment by DaniEASmith** [comment]
Hi, @JohanFreelancer9, this PR was moved to this other PR: https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness-pr/pull/10163, which was copyedited by me. I believe this PR can be closed now.

**Comment by JohanFreelancer9** [comment]
@scanum, for info.

**Pull request closed without merging**